### DISCUSSION TITLE
Is it possible to input multiple in vitro dissolution profiles at multiple pH simultaneously in the formulation building block

### DISCUSSION THREAD

**Discussion by SayeedMo** [Open-Systems-Pharmacology/Forum#1464] [Questions & Problems]
I have in vitro dissolution data at multiple pH for an delayed release tablets - at pH 6.8, 6.9, 7.0 and 7.1 - which demonstrates the percent of drug released from the delayed release tablets as a function of time.
I can see the option to input this in vitro dissolution data in the form of a table in the formulation building block - but I was looking for the options to specify at what pH the input data correlates to and also trying to understand if the software allows input of multiple in vitro dissolution profiles at all 4 different pH at the same time or do we have to create a separate simulation for dissolution data at each pH.
Kindly clarify
Thanks

**Comment by janakrw** [comment]
I have the same question. Approached OSP reps at PAGE 2024 and they suggested to post the question here. Anybody from development team to clarify? Thanks very much.

**Reply by Yuri05** [reply]
At the moment, it is not possible out of the box.
You can create a feature request for PK-Sim [here](https://github.com/Open-Systems-Pharmacology/PK-Sim/issues)
